etc/config.toml

```diff
@@ -21,6 +21,8 @@ time_to_min_bad = 1800  # 30 minutes
 [email]
 email_domain = 'organization.edu'  # Used for a generic email addr integration in integrations.py
 from_email = 'noreply+arbiter@organization.edu'
+smtp_passwd = '' # Used to login
+smtp_starttls = true # default false
```

Comment:
I believe most users do not use this, so it will probably be best to have these commented out in the example configuration file, along with a comment adding more information as to why it is used.

Reply:
Thanks for your reply. In my case, I must use SMTP login and enable STARTTLS to send emails to the email server.